runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1306,12 +1295,8 @@ def DeleteCommands()
   endif
 
   sign_unplace('TermDebug')
-  breakpoints = {}
-  breakpoint_locations = {}
-
```

Comment:
Those should be kept, no?

Reply:
I am not 100% sure, I may be missing something. :) What is the problem in leaving few script variable set when they will be reset anyway during the startup of the next Termdebug session? Do we really need to reset them to empty values at teardown? Why only these vars and not ALL the vars? (I am sure I am missing something though).

Reply:
However, I re-added them.

Reply:
True, I guess we don't neccessarily need this.